.gitignore

```diff
@@ -1,8 +1,4 @@
-# DS_Store
 .DS_Store
-*/.DS_Store
```

Comment:
I've seen it happen in other projects where there are redundant gitignore rules, especially for annoying files like .DS_Store. One `.DS_Store` matches all instances of .DS_Store in all subfolders, per the [gitignore docs](https://git-scm.com/docs/gitignore):

> A leading `**` followed by a slash means match in all directories. For example, `**/foo` matches file or directory `foo` anywhere, the same as pattern `foo`.